Examples/UIExplorer/NavigationExperimental/NavigationAnimatedExample.js

```diff
@@ -99,8 +100,10 @@ class NavigationAnimatedExample extends React.Component {
   _renderHeader(/*NavigationSceneRendererProps*/ props) {
     return (
       <NavigationHeader
-        {...props}
-        getTitle={state => state.key}
+        navigationProps={props}
```

Comment:
Your propTypes issue will probably be solved if you do `{...props}` here instead of adding `navigationProps`